### DISCUSSION TITLE
Customize Search Filter

### DISCUSSION THREAD

**Discussion by meostef** [microsoft-search/pnp-modern-search#1898] [Q&A]
Hi,

we would like to use the modern search for our Document Library with about 10k files. 

Is there a way to customize the Search Filter webpart? 
We are looking for a possibillity to split the filters. One part filter on the left, in the middle the search results, and one part filter on the right. 

Thanks! 

BR
meostef

**Comment by wobba** [comment]
@meostef You cannot connect two filter web parts to one result web part.
